config/routes.rb

```diff
@@ -1,6 +1,14 @@
 Rails.application.routes.draw do
+  get 'root/index'
+  namespace :api do
+    namespace :v1 do
+      resources :massages
+      get '/messages', to: 'massages#random_massage'
+    end
```

Comment:
- [ ] You did a great job applying the requested changes by the previous reviewer, however, I'm still getting a couple of errors, the first one is here in your routes, you are sending the `/messages` route to the `massages#random_massage` action which doesn't exist, it is named now `index` so you can safely remove line 6. 🤓 🔧 

- [ ] If you look closely, there's a typo in `resources :massages`, and also in the name of your controller, please either fix it or change the URL in your react app to fetch from `/api/v1/massages` instead! 🤓 🔧